sdk/Lusid.Sdk/Client/Configuration.cs

```diff
@@ -513,4 +513,4 @@ public static IReadableConfiguration MergeConfigurations(IReadableConfiguration
         }
         #endregion Static Members
     }
-}
+}
```

Comment:
Get this out of the diff just in case it causes us rebase problems as this is a generated file.

Reply:
👍